components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -208,6 +225,19 @@ export default class GalleryItem extends mixins(
     return [this.collectionId, this.id]
   }
 
+  get pageTitle(): string {
+    return `${this.nft.name || ''}`
```

Comment:
no template string needed here
```suggestion
    return this.nft?.name || ''
```

Reply:
oh no why did I copy this